tests/test_percolator.py

```diff
@@ -44,19 +44,25 @@
 from invenio_oaiserver.receivers import after_delete_oai_set, \
     after_insert_oai_set, after_update_oai_set
 
+from helpers import run_after_insert_oai_set
+
 
 def test_populate_oaisets(app, with_record_signals):
     """Populate OAISets."""
+    current_oaiserver.unregister_signals_oaiset()
```

Comment:
Is this one needed? It's not possible to simply remove the ``with_record_signals``-fixture?